_episodes/16-practices.md

```diff
@@ -228,6 +228,40 @@ Peak rule.
     While it has been criticized for not strongly predicting what's remembered,
     it's always worth trying to end a lesson on a high note.
 
+> ## Why Not a MOOC?
+>
+> Many learners find it difficult to get to a workshop,
+> either because there isn't one locally
+> or because it's difficult to schedule time around other commitments,
+> so why don't we create video recordings of the lessons
+> and offer the workshop as a MOOC (Massive Open Online Course)?
+>
+> The first answer is that we did in 2010-11,
+> but found the maintenance costs unsustainable.
+> Making a small change to a text file like this one only takes a few minutes.
+> but making *any* change to a video takes an hour or more.
+> In addition,
+> most people are much less comfortable recording themselves
+> than contributing written material.
+>
+> The second answer is that doing significantly outperforms watching.
+> A [recent paper by Koedinger et al]({{ site.github.url }}/files/papers/koedinger-doing-watching-2015.pdf)
+> estimated "...the learning benefit from extra doing (1 SD increase)
+> to be more than six times that of extra watching or reading."
```

Comment:
There's a lot going on in that paper.  Suggested clarification along these lines:

    The second answer is that doing significantly outperforms watching.  Specifically, a [recent paper by Koedinger et al]({{ site.github.url }}/files/papers/koedinger-doing-watching-2015.pdf) estimated "...the learning benefit from extra doing (1 SD increase) to be more than six times that of extra watching or reading."  *Doing*, in this case, refers to completing an interactive or mimetic task with feedback, while *benefit* refers to both *completion rates* and *overall performance*.